addons/rating/models/rating_mixin.py

```diff
@@ -156,7 +159,12 @@ def rating_get_grades(self, domain=None):
                                                 31-69%: Okay
                                                 70-100%: Great
         """
-        data = self._rating_get_repartition(domain=domain)
+        res_id_data = self._rating_get_repartition(domain=domain)
+        data = {}
+        for stats in res_id_data.values():
+            for stat, count in stats.items():
+                data[stat] = data.get(stat, 0) + count
+
         res = dict.fromkeys(['great', 'okay', 'bad'], 0)
         for key in data:
             grade = rating_data._rating_to_grade(key)
```

Comment:
Not sure I undestand what is done here. Also, it seems broken from a while as there is no `rating_data`